src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -857,39 +858,75 @@ describe("calcite-dropdown", () => {
     expect(await dropdownWrapper.isVisible()).toBe(false);
   });
 
+  it("is accessible", async () => {
+    accessible(dedent`
+      <calcite-dropdown>
+        <calcite-button slot="dropdown-trigger" id="trigger">Open dropdown</calcite-button>
+        <calcite-dropdown-group id="group-1" selection-mode="multi">
+          <calcite-dropdown-item id="item-1">
+            Dropdown Item Content
+          </calcite-dropdown-item>
+          <calcite-dropdown-item id="item-2" active>
+            Dropdown Item Content
+          </calcite-dropdown-item>
+          <calcite-dropdown-item id="item-3" active>
+            Dropdown Item Content
+          </calcite-dropdown-item>
+        </calcite-dropdown-group>
+        <calcite-dropdown-group id="group-2" selection-mode="single">
+          <calcite-dropdown-item id="item-4">
+            Dropdown Item Content
+          </calcite-dropdown-item>
+          <calcite-dropdown-item id="item-5" active>
+            Dropdown Item Content
+          </calcite-dropdown-item>
+        </calcite-dropdown-group>
+        <calcite-dropdown-group id="group-3" selection-mode="none">
+          <calcite-dropdown-item id="item-6">
+            Dropdown Item Content
+          </calcite-dropdown-item>
+          <calcite-dropdown-item id="item-7" href="google.com">
+            Dropdown Item Content
+          </calcite-dropdown-item>
+        </calcite-dropdown-group>
+      </calcite-dropdown>
+    `);
+  });
+
   it("correct role and aria properties are applied based on selection type", async () => {
     const page = await newE2EPage();
-    await page.setContent(`
-    <calcite-dropdown>
-    <calcite-button slot="dropdown-trigger" id="trigger">Open dropdown</calcite-button>
-    <calcite-dropdown-group id="group-1" selection-mode="multi">
-    <calcite-dropdown-item id="item-1">
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    <calcite-dropdown-item id="item-2" active>
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    <calcite-dropdown-item id="item-3" active>
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    </calcite-dropdown-group>
-    <calcite-dropdown-group id="group-2" selection-mode="single">
-    <calcite-dropdown-item id="item-4">
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    <calcite-dropdown-item id="item-5" active>
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    </calcite-dropdown-group>
-    <calcite-dropdown-group id="group-3" selection-mode="none">
-    <calcite-dropdown-item id="item-6">
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    <calcite-dropdown-item id="item-7" href="google.com">
-    Dropdown Item Content
-    </calcite-dropdown-item>
-    </calcite-dropdown-group>
-    </calcite-dropdown>`);
+    await page.setContent(dedent`
+      <calcite-dropdown>
```

Comment:
Nitpick: if this HTML is the same as above, storing them in a variable to share would clean up things a bit.